Source: <jieryn@gmail.com> in http://bugs.gentoo.org/show_bug.cgi?id=136843
Upstream: Yes, http://svn.debian.org/wsvn/pinfo/pinfo/trunk/src/?rev=302&sc=1
Reason: Fix off-by-one error

diff -Naur pinfo-0.6.9.old/src/parse_config.c pinfo-0.6.9/src/parse_config.c
--- pinfo-0.6.9.old/src/parse_config.c	2008-05-24 23:35:54.000000000 +0200
+++ pinfo-0.6.9/src/parse_config.c	2008-05-24 23:36:43.000000000 +0200
@@ -601,7 +601,7 @@
 		else
 			return 1;
 	}
-	else if (!strncmp(temp, "FTPVIEWER", 10))
+	else if (!strncmp(temp, "FTPVIEWER", 9))
 	{
 		temp = strtok(NULL, "\n");
 		if (temp)
